### PR TITLE
SUM-172431 add monitor playbook support

### PR DESCRIPTION
**Pull request by stevenxqli**
added optional string field for playbook in monitor terraform.

### CODE REVIEW

File: sumologic/resource_sumologic_monitors_library_monitor.go [hunk 1]
```diff
@@ -1099,6 +1106,7 @@ func resourceToMonitorsLibraryMonitor(d *schema.ResourceData) MonitorsLibraryMon
 		IsDisabled:         d.Get("is_disabled").(bool),
 		Status:             status,
 		GroupNotifications: d.Get("group_notifications").(bool),
+		Playbook:           d.Get("playbook").(string),
```
Comment (rsanbhad) [inline sumologic/resource_sumologic_monitors_library_monitor.go:1109]: please update the UTs to include this field (either in existing tests or new ones - but probably easier to add it to the existing tests)

**Review by rsanbhad — COMMENTED** [review]
some comments, overall looks straightforward - will leave approval to @kumar-avijit (would also recommend testing locally/manually with make install and some local terraform configuration for a monitor with and without a playbook against nite)

File: sumologic/sumologic_monitors_library_monitor.go [hunk 2]
```diff
@@ -136,6 +136,7 @@ type MonitorsLibraryMonitor struct {
 	IsDisabled         bool                  `json:"isDisabled"`
 	Status             []string              `json:"status"`
 	GroupNotifications bool                  `json:"groupNotifications"`
+	Playbook           string                `json:"playbook"`
```
[the remark this answers is not in the document]
Comment (stevenxqli) [reply sumologic/sumologic_monitors_library_monitor.go]: updated in ad992fe

File: sumologic/resource_sumologic_monitors_library_monitor.go (hunk 1 above)
Comment (stevenxqli) [reply sumologic/resource_sumologic_monitors_library_monitor.go:1109]: added acceptance test in ad992fe

**Comment by stevenxqli** [comment]
plox test

File: sumologic/sumologic_monitors_library_monitor.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (ndhar-sumo) [reply sumologic/sumologic_monitors_library_monitor.go]: yes `omitempty` was needed in my case, since I added an optional field.

**Review by rsanbhad — APPROVED** [review]
lgtm - also IIRC plox test doesn't work in this repo, you have to click the travis CI link and re-run from there - I've triggered the re-run now that Sean mentioned the TestAccUser errors should be fixed

**Pull request merged**